src/components/_pages/certificates/detail/index.tsx

```diff
@@ -1235,6 +1241,15 @@ export default function CertificateDetail() {
         [certLocations, locations],
     );
 
+    const defaultViewPort = useMemo(
```

Comment:
is this somehow related to certificates detail? if not and it can be reused elsewhere, then it would be better to extract somewhere else

Reply:
defaultViewPort is an optional prop that will be different for different flow charts which will be customized as per the UI output if the fitView prop in react flow is not sufficient .. so this will be slightly different for different flowcharts